rerun_py/rerun_sdk/rerun/sinks.py

```diff
@@ -308,8 +363,11 @@ def spawn(
         Specifies the [`rerun.RecordingStream`][] to use if `connect = True`.
         If left unspecified, defaults to the current active data recording, if there is one.
         See also: [`rerun.init`][], [`rerun.set_global_data_recording`][].
-    blueprint:
-        An optional blueprint to configure the UI.
+    default_blueprint
+        Optionally set a default blueprint to use for this application. If the application
+        already has an active blueprint, this blueprint won't become active until the user
+        clicks the "reset blueprint" button. If you want to replace the active blueprint
```

Comment:
What happens if the application _doesn't_ have an active blueprint? Will the new default blueprint become active? I'd assume yes, but it would be good to be explicit here imho